src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -729,6 +731,46 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
                 || WeightableLayerTransformation::isAsymmetricOnWeights(node, defaultPrecisions);
         });
 
+        lptPassConfig->set_callback<TransposeTransformation>([&](const_node_ptr& node) -> bool {
+            for (auto& user : node->get_users()) {
+                if (ov::is_type<ov::op::v13::ScaledDotProductAttention>(user))
+                    return true;
+            }
+
+            return false;
+        });
+
+        lptPassConfig->set_callback<MarkupPrecisions>([](const_node_ptr& node) -> bool {
+            return ov::is_type<ov::opset1::Multiply>(node) && !MultiplyToGroupConvolutionTransformation::canBeTransformedToGroupConvolution(node);
+        });
+        lptPassConfig->set_callback<ConvolutionBackpropDataTransformation>([&defaultPrecisions](const_node_ptr& node) -> bool {
+            return LayerTransformation::isAsymmetricQuantization(node, defaultPrecisions) ||
+                WeightableLayerTransformation::isAsymmetricOnWeights(node, defaultPrecisions);
+        });
```

Comment:
This callback overrides the existing one, dropping the handling of some additional parameters:
https://github.com/openvinotoolkit/openvino/blob/f9406ad17c8ae5f6985d8baefd5e726a5c913898/src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp#L696